src/base/simulation.jl

```diff
@@ -134,15 +152,15 @@ end
 
 function _attach_inner_vars!(
     device::PSY.DynamicGenerator,
-    ::Type{T} = Real,
+    ::Type{T} = Float64,
```

Comment:
Why going back to Float64 here? We required this change to work with implicit Julia solvers.

Reply:
I think this is old, will modify it